src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/tests/updateRouterAttributesFromView.test.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import updateRouterAttributesFromView from '../updateRouterAttributesFromView';
 import viewRegistry from '../registries/viewRegistry';
-import type {Route} from '../../../services/Router';
+import Route from '../../../services/Router/Route';
```

Comment:
Use only `services/Router` instead of the direct file to import from another package.